ocp_install_env.sh

```diff
@@ -98,10 +98,10 @@ EOF
 }
 
 function additional_trust_bundle() {
-    if [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ]; then
+    if [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ] && [ -z $MIRROR_IMAGES ]; then
```

Comment:
This still won't work when MIRROR_IMAGES is set (this is a very common option) - image_mirror_config writes the mirror config and it ends like:

```
additionalTrustBundle: |
${INDENTED_CERT}
```

So here we need to not write the `additionalTrustBundle: |` part when MIRROR_IMAGES is set, something like:

```
    if [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ] && [ -z "$MIRROR_IMAGES" ]; then
cat <<EOF
additionalTrustBundle: |
    $(echo ${ADDITIONAL_TRUST_BUNDLE} | awk '{ print " ", $0 }')
EOF
    elif [ ! -z "$ADDITIONAL_TRUST_BUNDLE" ] && [ ! -z "$MIRROR_IMAGES" ]; then
cat <<EOF
    $(echo ${ADDITIONAL_TRUST_BUNDLE} | awk '{ print " ", $0 }')
EOF
    fi
```

Reply:
@hardys With this code you actually say that  `$MIRROR_IMAGES` is not affecting at all.

By looking at [image_mirror_config function](https://github.com/openshift-metal3/dev-scripts/blob/master/utils.sh#L316-L343) we are skipping the whole function if `$MIRROR_IMAGES` is empty.
On my function we are adding `additionalTrustBundle` iff `$MIRROR_IMAGES` is empty. Therefore if `$MIRROR_IMAGES` is not empty, my function will be skipped & `image_mirror_config` won't.

Reply:
@hardys Understood, done.

Reply:
Thanks!